README.md

```diff
@@ -18,27 +17,37 @@ The Superchain token list is used as the source of truth for the [Optimism bridg
+These automated checks take place as part of the `Validate PR` check.
+Some issues raised by CI will trigger an error and must be resolved before your PR will be approved.
 These issues are marked below as "auto-reject" issues.
-Other issues will not immediately cause an error, but will require that a reviewer manually review your PR before it can be merged.
+Other issues will surface a warning, and will require a manual review from a reviewer.
 These issues are marked below as "requires manual review".
 
 - Given tokens actually exist on all specified chains (auto-reject)
 - L1 tokens are verified on Etherscan (auto-reject)
-- Description exists and is under 150 characters (auto-reject)
-- Website exists and loads properly (auto-reject)
+- Description is under 1000 characters (auto-reject)
 - Token `name`, `symbol`, and `decimals` matches on-chain data (auto-reject)
   - If `overrides` are used (requires manual review)
-- L2 token was deployed by the StandardTokenFactory (requires manual review)
-- Ethereum token listed on the [CoinGecko Token List](https://tokenlists.org/token-list?url=https://tokens.coingecko.com/uniswap/all.json)
+- L2 token was deployed by the [StandardTokenFactory](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/contracts/L2/messaging/L2StandardTokenFactory.sol) or is an [L2StandardERC20](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/contracts/standards/L2StandardERC20.sol) token that uses the standard L2 bridge address (`0x4200000000000000000000000000000000000010`) (requires manual review)
+- Ethereum token listed on the [CoinGecko Token List](https://tokenlists.org/token-list?url=https://tokens.coingecko.com/uniswap/all.json)(requires manual review)
   - *Why CoinGecko? CoinGecko's token list updates every hour which means we get token list updates very quickly. CoinGecko also uses an in-depth [listing criteria](https://www.coingecko.com/en/methodology).*
 
+#### Debugging Automated checks failures
+If your automated checks failed, you can see the reason for the failure by downloading `validation-artifacts.zip`, unzipping it and opening the `validation_results.txt` file. To locate the `validation-artifacts` follow these steps:
+
+1. Click on the `Details` link for `Validate PR` check in your PR
+2. Click `Summary` in the left panel
+3. Find the section on the page labeled `Artifacts` and click on `validation-artifacts`
+4. After download of `validation-artifacts.zip`, unzip it and open `validation_results.txt`
+
+If you make changes and need to run the validation check again, you will need to wait for a reviewer to approve the checks to run again. However, if you do not want to wait for a reviewer to approve the checks to run again to see if the failures have been resolved, you can run the validation checks locally by running:
+```
+tsx ./bin/cli.ts validate --datadir ./data --tokens <data folder name (e.g. ETH)>
```

Comment:
```suggestion
npx tsx ./bin/cli.ts validate --datadir ./data --tokens <data folder name (e.g. ETH)>
```

Reply:
This is a safer command to suggest since it works even if they haven't installed tsx globally

Reply:
nice, good suggestion